src/components/portrait/portrait.component.js

```diff
@@ -21,6 +26,10 @@ class Portrait extends React.Component {
     }
   }
 
+  marginProps() {
```

Comment:
I would change the name to `getMarginProps` so it is more verbose about what the property does